docs/validators/staking-v4-revised.md

```diff
@@ -0,0 +1,253 @@
+These selected nodes remain in the Auction List for one epoch. 
+
+The Auction List was devised with security in mind , randomly selecting nodes with the point being to avoid colluding, 
+giving Staking Providers a fair change for validating and also keeps the Shards healthy with shuffling nodes every epoch.
+
+There are three different status titles for nodes in the Auction List, Qualified, Danger and Not Qualified.
+
+![](/validators/stakingV4/AuctionList.png)
+
+
+## Epoch Start
+
+Every Epoch Start 320 Nodes are randomly shuffled out from the consensus and moved to the Auction List.
+
+This includes jailed nodes, but the mechanism compensates for each jailed node by selecting the same number of nodes that are not jailed.  
+
+
+## Qualified Status
+
+A Node gets the Qualified status if the Node NQT calculated for it, from the stake of Staking Provider and top-up, is the same or higher as compared to the Network NQT.
+
+![](/validators/stakingV4/QualifiedSimple.png)
+
+![](/validators/stakingV4/QualifiedDetail.png)
+
+
+## Danger Status 
+
+A Node gets the Danger status if the Node NQT calculated for it, from the stake of Staking Provider and top-up, is dangerously close or exactly the same as the Network NQT.
+
+![](/validators/stakingV4/DangerSimple.png)
+
+![](/validators/stakingV4/DangerDetail.png)
+
+
+## Not Qualified Status
+
+A Node gets the Not Qualified status when the Node NQT calculated for it, from the stake of Staking Provider and top-up,
+is lower as compared to the Network NQT, the active period for the Validator Node.
+
+![](/validators/stakingV4/NotQualifiedSimple.png)
+
+![](/validators/stakingV4/NotQualifiedDetail.png)
+
+
+## Waiting List
+
+The Waiting List represents the list of nodes that have been Qualified from the Auction List.
+All the nodes that have passed the Qualified Status will then go to the Waiting List, they will remain in the Waiting List for 4 epochs which represents roughly four days.
+
+After the 4 epochs, these nodes are moved to participate in the consensus for validating.
+As a side note, even if the Network NQT changes in the 4 waiting Epochs the already selected nodes will still participate in the consensus even if they would presumably have lower Node NQT after the aforementioned 4 epochs in the Waiting List.
+
+
+## Waiting Status
+
+This Status means that the Validator Node has been Qualified from the Auction List and is in the 4 Epoch waiting time before joining the consensus to start validating.
+
+![](/validators/stakingV4/WaitingDetail.png)
+
+
+## Eligible Status
+
+This Status means that the Validator Node can validate or propose blocks in the current Epoch. It can be part of the Consensus in certain rounds.
+
+![](/validators/stakingV4/EligibleDetail.png)
+
+
+## Automatic Node Qualification
+
+This mechanism automatically distributes a Staking Provider's total top-up amount to the Validator Nodes they own.
```

Comment:
Why do we need this chapter? Feels like it adds a lot of confusion. Cand be interpreted as if the system is "playing" with your own stake.

Reply:
cc @schimih